src/geonovum/seo.js

```diff
@@ -13,42 +13,44 @@ export function run(conf, doc, cb) {
       conf.shortName +
       "/"
     : null;
-  switch (conf.canonicalURI.toLowerCase()) {
-    case "wv":
-      if (conf.edDraftURI) {
-        conf.canonicalURI = new URL(conf.edDraftURI, doc.location).href;
-      } else {
-        pub(
-          "warn",
-          "Canonical URI staat op WV, " +
-            "maar er is geen edDraftURI geconfigureerd"
-        );
-        conf.canonicalURI = null;
-      }
-      break;
-    case "def":
-      if (trLatestUri) {
-        conf.canonicalURI = trLatestUri;
-      } else {
-        pub(
-          "warn",
-          "Canonical URI staat op DEF" +
-            "maar er is geen shortName geconfigureerd"
-        );
-        conf.canonicalURI = null;
-      }
-      break;
-    default:
-      if (conf.canonicalURI) {
-        try {
-          conf.canonicalURI = new URL(conf.canonicalURI, doc.location).href;
-        } catch (err) {
-          pub("warn", "CanonicalURI is an invalid URL: " + err.message);
+  if (conf.canonicalURI) {
+    switch (conf.canonicalURI.toLowerCase()) {
+      case "WV":
```

Comment:
Note, these will never match, because it's expecting lower case

Reply:
You're right